### PR TITLE
Add terms and condition modal

### PR DESCRIPTION
**Pull request by ronaldsg20**

### CODE REVIEW

**Review by alexjavabraz — DISMISSED** [review]
LGTM

**Review by leoiovlabs — DISMISSED** [review]
lgtm

File: src/common/components/common/TermsDialog.vue [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        </v-row>
+        <v-row class="mx-0 mt-6 mb-3 d-flex justify-center">
+          <h2>Terms and conditions</h2>
+        </v-row>
+        <v-row>
+          <v-container class="terms-txt mx-15 my-6 pa-10" @scroll="onScroll">
+            <v-row>
+              <h3>The standard Lorem Ipsum passage, used since the 1500s</h3>
+              <p>
+                "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
```
Comment (alexjavabraz) [inline src/common/components/common/TermsDialog.vue:16]: Add the text here

**Review by lserra-iov — APPROVED** [review]

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Pull request merged**